### PR TITLE
COMMERCE-10315 fixed js

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-10315

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-asset-categories-navigation-web/src/main/resources/META-INF/resources/configuration.jsp [hunk 1]
```diff
@@ -129,13 +129,17 @@ if (assetVocabulary != null) {
...
-			preferencesRootAssetCategoryId.val(
-				form.elements[assetCategoryIdsKey].value
-			);
+			for (const assetCategoryId of assetCategoryIdsKey) {
+				if (form.elements[assetCategoryId].value) {
+					preferencesRootAssetCategoryId.val(
+						form.elements[assetCategoryId].value);
+					break;
+				}
+			}
```
Comment (kresimir-coko) [inline modules/apps/commerce/commerce-product-asset-categories-navigation-web/src/main/resources/META-INF/resources/configuration.jsp:142]: We do not want to use `for...of` loops.

[Here's one file that's overriding the rule](https://github.com/liferay/liferay-portal/blob/master/modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/forms/formsHelper.js#L28-L42), which we don't want to do.

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by iovaneale** [comment]
Sent to #3249.

@andrea-ale-sbarra please close this one.

**Pull request closed without merging**